lib/cocina_display/cocina_record.rb

```diff
@@ -51,13 +51,12 @@ def modified_time
     end
 
     # SDR content type of the object.
-    # @note {RelatedResource}s may not have a content type.
```

Comment:
This comment is from when `RelatedResource` inherited from this class – it no longer does, and thus doesn't have this method at all.